ui/app/templates/components/http-requests-bar-chart.hbs

```diff
@@ -10,9 +10,9 @@
   <g  class="gridlines"></g>
   <g class="x-axis"></g>
   <g class="y-axis"></g>
-  <g clip-path="url(#bars-container-{{this.elementId}})">
```

Comment:
I like how `this.` is more explicit and you know that it's not a helper that way. Not something you need to change back, but something we should discuss and determine what we want our convention to be. The named args RFC is also related to this: https://github.com/emberjs/rfcs/blob/master/text/0276-named-args.md

Reply:
oh, i like the `this.elementId` and named args pattern better too. i've had difficulty discerning whether something is a helper, attribute, or block variable as i've been learning ember and this makes things much clearer. i'll change it back, and i'm all for using named args as a convention going forward!